horace_core/sqw/PixelData/@PixelData/PixelData.m

```diff
@@ -104,34 +105,30 @@
         page_number_ = 1;  % the index of the currently loaded page
         raw_data_ = zeros(PixelData.DEFAULT_NUM_PIX_FIELDS, 0);  % the underlying data cached in the object
         tmp_io_handler_;  % a PixelTmpFileHandler object that handles reading/writing of tmp files
-        pix_range_=PixelData.EMPTY_RANGE_; % range of pixels in Crystal cartesian coordinate system
+        pix_range_=PixelData.EMPTY_RANGE_; % range of pixels in Crystal Cartesian coordinate system
     end
     
     properties (Constant)
         DATA_POINT_SIZE = 8;  % num bytes in a double
         DEFAULT_NUM_PIX_FIELDS = 9;
         DEFAULT_PAGE_SIZE = realmax;  % this gives no paging by default
+    end
+    properties (Constant,Hidden)
         % the range, an empty pixel class has
         EMPTY_RANGE_ = [inf,inf,inf,inf;-inf,-inf,-inf,-inf];
     end
     
-    properties (Dependent, Access=private)
-        data_;  % points to raw_data_ but with a layer of validation for setting correct array sizes
-        
-        pix_position_;  % the pixel index in the file of the first pixel in the cache
-    end
-    
     properties (Dependent)
-        % The 1st dimension of the crystal cartesian orientation (1 x n array) [A^-1]
+        % The 1st dimension of the crystal Cartesian orientation (1 x n array) [A^-1]
```

Comment:
being very picky here - you have corrected "Crystal cartesian" to "Crystal Cartesian" above. SHouldnt you be making the same correction here i.e. capitalise crystal?

Reply:
may be except Cartesian is highlighted by spell-checker an crystal is not. I pay attention to red lines in spell-checker but not to text itself.